spec/components/task_list/view_spec.rb

```diff
@@ -0,0 +1,68 @@
+require "rails_helper"
+
+RSpec.describe TaskList::View do
+  alias_method :component, :page
```

Comment:
Love this!!! `page` in this context is a Capybara object of just the rendered component. so to improve the test @darokel helped me alias this so that its obvious that its just the component being tested. We will need to see if theres some way to make this a global option for component tests

Reply:
Ya, when I saw the `have_selector` stuff below it made me wonder if we can't use `SitePrism` here.

Reply:
100% thats is the dream, to be able to use siteprism with component tests